src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -156,7 +199,13 @@ public class PullRequestHandler : IHttpHandler
         return success;
     }
 
-    private bool SyncToPr(uint pr, StringBuilder executionResult)
+    private string GetRepoPath(uint repoId)
```

Comment:
I'm willing to take named repos (as opposed to numerical) as an item to discuss for later, which probably will tie into a discussion on how we get Helix to work for private runs. 

Given the discussions we've had and scenarios offered have always stated clearly that we will use numbered repos only, I stand by my decision to implement as shown here. 

Unless there's something really wrong with how this is implemented for numbered repos, I propose leaving this discussion for later and taking this as an item to discuss.